index.js

```diff
@@ -0,0 +1,16 @@
+const redBlock = document.querySelector(".block--red");
+const blueBlock = document.querySelector(".block--blue");
+const greenBlock = document.querySelector(".block--green");
+const pinkBlock = document.querySelector(".block--pink");
+const grayBlock = document.querySelector(".block--gray");
+
+let grayBlockTraveler = grayBlock.style.marginLeft;
```

Comment:
Instead of manually adding `addEventListener` to each element, you can use `querySelectorAll` to get an array of the elements and use `forEach` to iterate through each element to add the `addEventListener` that way.